src/components/main/map/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, {
+  FC,
+  ReactNode,
+  useRef,
+  useEffect,
+  useState,
+  isValidElement,
+  cloneElement,
+  Children,
+} from "react";
+import { useDeepCompareEffectForMaps } from "@/hooks/maps";
+import { mapStyles } from "./mapStyles";
+import styles from "./Map.module.css";
+
+interface MapProps extends google.maps.MapOptions {
+  onClick?: (e: google.maps.MapMouseEvent) => void;
+  children?: ReactNode;
+}
+
+const Map: FC<MapProps> = ({ children, onClick, ...options }) => {
```

Comment:
```suggestion
const Map: FC<MapProps> = ({ children, onClick, ...props }) => {
```
Usually in this case "props" is used. WDTY

Reply:
Makes sense, updated!